DEVX-5668 Add image download for Android (#3734)
Nexmo/nexmo-developer: _documentation/en/client-sdk/in-app-messaging/guides/send-receive-image.md

source: _tutorials_tabbed_content/client-sdk/guides/messaging/send-images
```

## Receive an Image URL

A `image` conversation event will be received when a member sends an image to a conversation:

```tabbed_content
source: _tutorials_tabbed_content/client-sdk/guides/messaging/receive-image-urls
```

## Download images

> To download image you should add JWT to the image retrieval request. JWT is passed as an Authorization header (`Authorization: Bearer JWT` format). This is the JWT that was used to login the user. 

Reviewer: ```suggestion
To download an image from Vonage you need to add JWT to the image retrieval request. The JWT is passed as an Authorization header (`Authorization: Bearer <JWT>` format). This is the JWT that was used to log in the user. 
```

> Various image libraries are handling request headers differently, so below you will fin example for the most popular libraries:

Reviewer: ```suggestion
Various image libraries are handling request headers differently, so below you will full example for the most popular libraries:
```
